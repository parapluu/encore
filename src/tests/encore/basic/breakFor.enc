passive class Phuc
  def phuc() : void
    for i <- [0..10] do
      print ("{} ",i)
      if i>5 then
        break
        print ("Never show")
      end
    end
  end
end

class Main
  def main() : void
    for i <- [0..10] do
      print ("{} ",i)
      if i>5 then
        break
        print ("Never show")
      end
    end
    print ("Done!\n")
    (new Phuc).phuc()
    print ("Done!\n")
  end
end
