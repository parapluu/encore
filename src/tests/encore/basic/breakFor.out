0 1 2 3 4 5 6 Done!
0 1 2 3 4 5 6 Done!
